<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardDialogs</class>
 <widget class="QWidget" name="StandardDialogs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandardDialogs</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>681</width>
     <height>401</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>输入对话框</string>
    </attribute>
    <widget class="QPushButton" name="btn_getint">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getInt()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getdouble">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getDouble()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getitem">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getItem</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_gettext">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getText()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_getmtext">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>211</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getMultiText()</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_input">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>颜色对话框</string>
    </attribute>
    <widget class="QPushButton" name="btn_getcolor">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>89</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>getColor()</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_alpha">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>是否</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>字体对话</string>
    </attribute>
    <widget class="QPushButton" name="btn_getfont">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getFonts()</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_font">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>160</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>您好，QT</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>文件对话框</string>
    </attribute>
    <widget class="QLineEdit" name="line_path">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>351</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dir">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getExistDir()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_file">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>140</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getOpenFileName()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_files">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getOpenFileNames()</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_savefile">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>getSaveFileName()</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>消息对</string>
    </attribute>
    <widget class="QPushButton" name="btn_critical">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>critical</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_info">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>information</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_warning">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>warning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_question">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>question</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_msg">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>showMsg</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>进度对话框</string>
    </attribute>
    <widget class="QPushButton" name="btn_progress">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>进度</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>向导对话框</string>
    </attribute>
    <widget class="QPushButton" name="btn_wizard">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>弹出向导</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
